.dep-versions

```diff
@@ -5,3 +5,6 @@ llvm=cd9a641613eddf25d4b25eaa96b2c393d401d42c
 enzyme=1beb98b51442d50652eaa3ffb9574f4720d611f1
 
 # Always remove custom PL/LQ versions before release.
+
+lightning=0.36.0.dev30
```

Comment:
Could you please update this to dev32 to include some of the decomposition and serialization fixes for LQ and LK?
 
```suggestion
lightning=0.36.0-dev32
```